subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/rules/VariantFilesMetadataRulesIntegrationTest.groovy

```diff
@@ -513,11 +513,12 @@ class VariantFilesMetadataRulesIntegrationTest extends AbstractModuleDependencyR
     def "can add variants for ivy - #usageAttribute - honors conf based excludes "() {
         // through this, we opt-into variant aware dependency management for a pure ivy module
         given:
+        System.setProperty('org.gradle.integtest.force.realize.metadata', 'true')
```

Comment:
Hi @ljacomet ,

Is this necessary or was it only for some local testing ? :)

Reply:
Ah thanks - removing!